www/__tests__/customMetricsHelper.test.ts

```diff
@@ -41,12 +52,13 @@ it('gets the custom mets', async () => {
 it('gets the custom footprint', async () => {
   const appConfig = await getConfig();
   await initCustomDatasetHelper(appConfig);
+  //numbers from fakeLabels.json
   expect(getCustomFootprint()).toMatchObject({
-    walk: expect.any(Number),
-    bike: expect.any(Number),
-    bikeshare: expect.any(Number),
-    'e-bike': expect.any(Number),
-    scootershare: expect.any(Number),
-    drove_alone: expect.any(Number),
+    walk: 0,
+    bike: 0,
+    bikeshare: 0,
+    'e-bike': 0.00728,
```

Comment:
If I recall correctly, that was because of the `-` -- when I get the chance to work on this and address the rest of review I'll double check it's needed!

Reply:
The quotes prevent a syntax error, I added them to the rest for consistency, but prettier removed them so I think it has to be formatted like this